app/uploaders/asset_uploader.rb

```diff
@@ -1,25 +1,52 @@
 class AssetUploader < CarrierWave::Uploader::Base
+  ALLOWED_EXTENSIONS = %w(jpg jpeg gif png pdf)
+  IMAGE_EXTENSIONS = %w(jpg jpeg gif png)
+
   include CarrierWave::MiniMagick
```

Comment:
Do we need @cnuber to make sure imagemagick is installed on Exercises deployments?